### ISSUE TITLE
Add "intercept require :hotpot" option

### ISSUE THREAD

**Issue by rktjmp** [rktjmp/hotpot.nvim#28]
Re: #27 where the cljlib must be patched so it's `require :fennel` calls can work without including fennel somewhere else.

It would be nice if Hotpot could _optionally_ be instructed to intercept require calls for fennel and provide it's already included version.

```
require("hotpot").setup({
  intercept_require_fennel = true
})
```

**Comment by rktjmp** [comment]
https://github.com/rktjmp/hotpot.nvim/pull/30

**Comment by rktjmp** [comment]
Fixed in #30
